changelog/1379.doc.rst

```diff
@@ -0,0 +1 @@
+Fixes the docstring of the coupling_parameter function in plasmapy/formulary/collisions.py.
```

Comment:
```suggestion
Fixed the docstring of the `~plasmapy.formulary.collisions.coupling_parameter`
function in :file:`plasmapy/formulary/collisions.py`\ .
```
The single back ticks enclosing the whole path make our online docs link to the function, and the `:file:` role formats the text in a style to indicate that it is a file.